app/src/main/java/org/wikipedia/main/MainFragment.java

```diff
@@ -137,6 +138,9 @@ public static MainFragment newInstance() {
             if (getCurrentFragment() instanceof FeedFragment && item.getOrder() == 0) {
                 ((FeedFragment) getCurrentFragment()).scrollToTop();
             }
+            if (getCurrentFragment() instanceof HistoryFragment && item.getOrder() == NavTab.SEARCH.code()) {
+                openSearchActivity(NAV_MENU, null);
```

Comment:
Can we add `return` below the `openSearchActivity()` to prevent unexpected action of scrolling to the top in the history page?